controllers/ingressnodefirewall_controller.go

```diff
@@ -362,10 +367,7 @@ func (r *IngressNodeFirewallReconciler) buildNodeStates(
 }
 
 func isLegalInterfaceName(iface string) bool {
```

Comment:
nit, but whatever you prefer.

Perhaps just write this as:
~~~
var isLegalInterfaceName = func(iface string) bool {
~~~

That way, it's already a variable and line 50 isn't needed

Reply:
removed that code